test/check-machines-lifecycle

```diff
@@ -248,6 +248,13 @@ class TestMachinesLifecycle(VirtualMachinesCase):
         b.click("#rename-dialog-confirm")
         b.wait_in_text(".pf-c-modal-box__footer .pf-c-alert.pf-m-danger", "Can't rename domain to itself")
 
+        self.goToVmPage("new")
+        self.performAction("new", "rename")
+
+        b.set_input_text("#rename-dialog-new-name", "test%")
+        b.click("#rename-dialog-confirm")
+        b.wait(lambda: "vm?name=test%25&connection=system" in b.eval_js("window.location.href"))
```

Comment:
Ah, that's the test case for the first commit for correct URL encoding. I was about to complain there :grin: 

However, can you please ensure that the page actually works? I.e. check some expected element there. If the link would lead to the void or a page crash, that test case would still succeed, but we need to know that parsing and evaluating the URL actually works as intended.